src/bp/nlu-core/intents/base-intent-classifier.ts

```diff
@@ -28,30 +27,7 @@ export class BaseIntentClassifier implements IntentClassifier {
 
     const customEntities = this._getCustomEntitiesNames(list_entities, pattern_entities)
 
-    const noneUtts = _.chain(intents)
```

Comment:
This intent clf, is no longer "aware" of the concept of none. It must choose between the labels it has seen at training time.